memgpt/cli/cli.py

```diff
@@ -541,6 +542,7 @@ def run(
     agent_state = ms.get_agent(agent_name=agent, user_id=user.id) if agent else None
     human = human if human else config.human
     persona = persona if persona else config.persona
+    system_prompt = system if system else (agent_state.system if agent_state else None)
```

Comment:
@sarahwooders not totally sure about this line - is there somewhere (inside of `cli.py`) where the system prompt may have been specified?

probably not right, since we stripped presets?

Reply:
also maybe this needs to come out of `agent_state.memory.system`?

for example - if the system prompt had been modified previously?

https://github.com/cpacker/MemGPT/blob/main/memgpt/agent.py#L1051-L1055

Reply:
nvm i refactored - see comments below